ir/src/constraints/graph.rs

```diff
@@ -108,13 +108,18 @@ impl AlgebraicGraph {
                 default_domain,
             ),
             Expression::Rand(index) => {
-                // The constraint target for random values defaults to the second (auxiliary) trace
-                // segment.
+                // constraint target for random values defaults to the second trace segment.
```

Comment:
nit: I would probably leave this comment as it was, since we still have lots of references to "auxiliary" throughout the code